src/Model/Profile.php

```diff
@@ -1117,7 +1117,16 @@ public static function openWebAuthInit($token)
 		$_SESSION['visitor_handle'] = $visitor['addr'];
 		$_SESSION['visitor_home'] = $visitor['url'];
 		$_SESSION['my_url'] = $visitor['url'];
+		$_SESSION['remote'] = [];
 
```

Comment:
can we insert a `@todo` that this is a quick fix and that is planed to fix it in the functions which rely on `$_SESSION['remote']`?

Reply:
What do you mean it's a quick fix?

Reply:
I really don't like the way we are currently checking this session variable in the mentioned places. I would like to replace this with a single function call. And then